modules/db/src/main/scala/gem/dao/UserDao.scala

```diff
@@ -13,29 +13,10 @@ object UserDao {
     select("root")
 
   def select(id: String): ConnectionIO[User[Nothing]] =
-    sql"""
-      SELECT id, first, last, email, staff
-      FROM gem_user
-      WHERE id = $id
-    """.query[(String, String, String, String, Boolean)]
-       .unique.map { case (i, f, l, e, s) =>
-         User(i, f, l, e, s, Map.empty)
-       }
+    Statements.select(id).unique
```

Comment:
There is a pattern where you have moved the sql from the `Dao` level to an internal `Statements` object. Any reason for that or is it just a style change?